core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -140,6 +175,10 @@ export class CreateServiceComponent implements OnInit {
 
  // function for opening and closing create service popup
   closeCreateService(serviceRequest){
+    // this.typeofserviceSelected = false;
```

Comment:
Remove comments everywhere if not needed.

Reply:
REmoved